lib/platform/github/index.ts

```diff
@@ -1531,7 +1527,7 @@ async function deleteComment(commentId: number): Promise<void> {
   );
 }
 
-async function getComments(issueNo: number): Promise<Comment[]> {
+export async function getComments(issueNo: number): Promise<Comment[]> {
```

Comment:
We need to export the `getComments` method because we want to find out if a automerge comment already exists: https://github.com/renovatebot/renovate/pull/6002/files#diff-5a59e8794d3cba4d86a62354df383482R435